core/controllers/acl_decorators.py

```diff
@@ -1600,19 +1600,19 @@ def test_can_voiceover(self, exploration_id, **kwargs):
     return test_can_voiceover
 
 
-def can_manage_voice_artist(handler):
-    """Decorator to check whether the user can manage voice artist.
+def can_add_voice_artist(handler):
+    """Decorator to check whether the user add manage voice artist to activity.
```

Comment:
add manage --> can add

to activity --> to the given activity

Reply:
Done.

Reply:
Done.